app/src/androidTest/java/com/pajato/android/gamechat/ApplicationTest.java

```diff
@@ -48,70 +47,129 @@
 
     /** Ensure that the chat panel is being displayed. */
     @Test public void testChatPaneIsVisible() {
-        onView(withId(R.id.chat_pane)).check(matches(isDisplayed()));
+        onView(withId(R.id.chat_pane))
```

Comment:
Why two lines here?  My mantra is that if one concise line is as readable as two then less is better.